copyandpush/copyandpush.js

```diff
@@ -1,9 +1,6 @@
 const copyAndPush = (array, x) => {
   //fucntion needs to pass in an array of numbers and return a new array with the numbers in original array and a new number added to the end
-  const originalArrayCopy = array;
-  const newItem = x;
-
-  const newArray = [...originalArrayCopy, newItem];
+  const newArray = [...array, x];
```

Comment:
This is beautiful - I don't know if you've heard the term 'code golf,' but it usually refers to the sort of process I'm seeing you do here. It's basically seeing just how few lines of code you can manage to write something in (sometimes to the point of absurdity/obscurity, haha).

Reply:
Also, if you wanted to be even shorter about it you could do it in one line of code I think...
```js
return [...array, x];
```